### ISSUE TITLE
[💡 FEATURE REQUEST]: Statsd support for metrics

### ISSUE THREAD

**Issue by cv65kr** [temporalio/roadrunner-temporal#265]
### Plugin

Temporal

### I have an idea!

Since Temporal can supports statsd and prometheus via Tally. Would be nice to have the same feature for metrics client in Roadrunner.

**Comment by rustatian** [comment]
Hey @cv65kr 👋
RR already supports all temporal metrics: https://github.com/roadrunner-server/roadrunner/blob/master/.rr.yaml#L193

**Comment by cv65kr** [comment]
Hi @rustatian 
But as far I see in code, there are can be emitting only for prometheus.
Temporal itself supports multiple clients - https://docs.temporal.io/references/configuration/#statsd

**Comment by rustatian** [comment]
> Hi @rustatian But as far I see in code, there are can be emitting only for prometheus. Temporal itself supports multiple clients - https://docs.temporal.io/references/configuration/#statsd

Yeah, you're right. I thought that you didn't know about Prometheus in the RR. So yeah, we'll need to have identical providers here.
